nginx-app/resources/config.rb

```diff
@@ -9,3 +9,5 @@
 attribute :nginx_user, :kind_of => String
 attribute :nginx_group, :kind_of => String
 attribute :error_log, :kind_of => String
+attribute :error_echo_port, :kind_of => String
+attribute :access_echo_port, :kind_of => String
```

Comment:
these probably are not required since I'm not using them, I was trying to understand how these work but could not make it happen